src/GeositeFramework/js/Export.js

```diff
@@ -105,6 +129,20 @@ require(['use!Geosite'],
                                 .wrapAll("<div class='print-legend-coll'></div>");
                         });
 
+                    function afterPrintHandler() {
+                        _.delay(previewDeferred.resolve, 250);
+                    }
+
+                    $.when(alertDeferred).then(function () {
+                        if (!!window.onafterprint) {//ie, ff, safari?
+                            window.onafterprint = afterPrintHandler;
```

Comment:
Consider adding some documentation to explain why we have to perform browser sniffing. What is the difference between IE/FF/Safari and Chrome?